src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -268,10 +268,12 @@ export default class ResultsViewPage extends React.Component<
                             id={ResultsViewTab.MUTATIONS}
                             linkText="Mutations"
                         >
-                            <Mutations
-                                store={store}
-                                appStore={this.props.appStore}
-                            />
+                            {store.genes.isComplete && (
+                                <Mutations
```

Comment:
we seem to do this checking for loadind inside of tab components, not here.  ALso make sure we are showing loader